tests/Basic/Basic.h

```diff
@@ -762,3 +762,15 @@ struct CS_VALUE_TYPE ValueTypeArrays
 	char thirdValueTypeArray[ARRAY_LENGTH];
 	size_t size;
 };
+
+class DLL_API RefTypeToValTypeCheckBase
+{
+protected:
+	void doSmthProt() { }
```

Comment:
Please write all function bodies in the cpp file. The issue with inlined bodies is that they are not compiled by default on Linux and OS X and we want the tests to work there as well.